docs/source/quickstart/script.md

```diff
@@ -0,0 +1,262 @@
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --use_all_fields_at_inference | <class 'bool'>  | False           | Whether to require all fields in the serving       |
+|                               |                 |                 | signature of the SavedModel. If set to False, only |
+|                               |                 |                 | requires fields with required_only=True            |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --pad_sequence_at_inference   | <class 'bool'>  | False           | Whether to pad sequence at inference time. Used to |
+|                               |                 |                 | define the TFRecord serving signature in the       |
+|                               |                 |                 | SavedModel                                         |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --output_name                 | <class 'str'>   | relevance_score | Name of the output node of the model               |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --early_stopping_patience     | <class 'int'>   | 2               | How many epochs to wait before early stopping on   |
+|                               |                 |                 | metric degradation                                 |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --file_handler                | <class 'str'>   | local           | String specifying the file handler to be used.     |
+|                               |                 |                 | Should be one of FileHandler keys in               |
+|                               |                 |                 | ml4ir/base/config/keys.py                          |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --initialize_layers_dict      | <class 'str'>   | {}              | Dictionary of pretrained layers to be loaded.The   |
+|                               |                 |                 | key is the name of the layer to be assigned the    |
+|                               |                 |                 | pretrained weights.The value is the path to the    |
+|                               |                 |                 | pretrained weights.                                |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --freeze_layers_list          | <class 'str'>   | []              | List of layer names that are to be frozen instead  |
```

Comment:
why is the type not `list` ?

Reply:
These are command line arguments. So we read as string and parse in the script.